src/main/java/org/molgenis/mappers/VkglTableMapper.java

```diff
@@ -11,14 +11,15 @@ default String getId(String ref, String alt, String chr, String pos, String gene
     return Hasher.hash(id);
   }
 
-  default Map<String, String> getCorrectedRefAndAlt(String ref, String alt, String type,
-      int start) {
+  default Map<String, String> getCorrectedRefAndAlt(String ref, String alt, int start) {
     HashMap<String, String> corrected = new HashMap<>();
-    if (type.equals("sub") && ref.length() == 2 && alt.length() == 2) {
+
+    if (alt.length() > 1 && ref.length() > 1 && ref.charAt(0) == alt.charAt(0)) {
```

Comment:
Theoretically there can be more than one matching nucleotide I think?
AAT -> AATT 
If I read the code correctly this will result in AT -> ATT but should become T -> TT

Reply:
Or do we know this will simply never happen in the input data?

Reply:
At this point in the pipeline we will not have this kind of input because we already used a tool to normalise the data, this tool however puts an anchor in front of every variant, this is incorrect for indels and substitutions. This piece of code gets rid of the anchor for these kind of variants. Before it only corrected the substitutions.